internal/consts/consts.go

```diff
@@ -17,4 +17,5 @@ const (
 	EventSourceComponentCtxKey ctxKey = "event_source"
 	EventActorCtxKey           ctxKey = "event_actor"
 	TLSPeerCertificateCtxKey   ctxKey = "tls-peer-certificate"
+	OrganizationIDCtxKey       ctxKey = "organization-id"
```

Comment:
For 6 consts we have 3 different formats - underscores, hyphens, and camelCase.  Can you please set them all to hyphens as long as you're editing here?